src/components/open/CreateSafe/styles.module.css

```diff
@@ -0,0 +1,7 @@
+.back {
+  margin: var(--space-4) 0;
+}
+
+.title {
+  padding-bottom: var(--space-2);
+}
```

Comment:
These one-liners could be `my={4}` and `pb={2}` respectively.

Reply:
`Button` does not accept shorthands so I've added an `sx` to it.